go.mod

```diff
@@ -112,6 +112,8 @@ require (
 	github.com/c-bata/go-prompt v0.2.6 // indirect
 	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
 	github.com/cloudflare/circl v1.3.3 // indirect
+	github.com/confluentinc/ccloud-sdk-go-v2/ccp v0.0.1 // indirect
+	github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin v0.0.1 // indirect
```

Comment:
What's the difference between these dependencies? Looks like you forgot to run `go mod tidy`, btw!

Reply:
Ran go mod tidy in the end which removed the ccp dependency